.circleci/config.yml

```diff
@@ -192,7 +192,10 @@ jobs:
       - run: *bootstrap_old_test_suite
       - run:
           name: Run spider
-          command: cd src/api; bundle exec rake test:spider
+          command: |
+            cd src/api
+            export BENTO=1
+            bundle exec rake test:spider
```

Comment:
Hm, why do we run the spider test with bento?

Reply:
Because we never run the spider test with Bootstrap. Shall I adapt the spider test for it?

Reply:
What would you need to adapt?

Reply:
I'm interested in what needs to be adopted for that.

Reply:
not much, for sure the checking of the [logo](ttps://github.com/openSUSE/open-build-service/blob/master/src/api/test/functional/webui/spider_test.rb#L108)

Reply:
I'm all for having the spider test run with what we will use most. Not necessarily in this PR..